blazor/chart/how-to/synchronized-chart.md

```diff
@@ -0,0 +1,412 @@
+        </SfChart>
+    </div>
+
+</div>
+
+@code{
+
+    public SfChart? _chart1;
+    public SfChart? _chart2;
+    private bool isMouseMoveUpDetected;
+    private bool isTouch;
+    private bool isTapHold = false;
+    public double zoomFactor = 1;
+    public double zoomPosition = 0;
+
+    List<ToolbarItems> toolbarItems = new List<ToolbarItems>() { ToolbarItems.Pan, ToolbarItems.Reset };
+
+    protected override async Task OnInitializedAsync()
+    {
+        ChartPoints = await Http.GetFromJsonAsync<CurrencyRates[]>(NavigationManager.BaseUri + "./synchronized-data.json");
+    }
+
+    public class CurrencyRates
+    {
+        public DateTime USD { get; set; }
+        public double EUR { get; set; }
+        public double JPY { get; set; }
+        public double SGD { get; set; }
+        public double INR { get; set; }
+    }
+
+    public CurrencyRates[] ChartPoints { get; set; } = new CurrencyRates[] { };     
+
+    public void ChartMouseDown(ChartMouseEventArgs changeEventArgs)
+    {
+        isMouseMoveUpDetected = false;
+        isTouch = true;
+    }
+
+    public void OnMouseEventChart1(ChartMouseEventArgs args)
+    {
+        _chart2.ShowTooltip(args.MouseX, args.MouseY, false);         
+    }
+
+    public void OnMouseEventChart2(ChartMouseEventArgs args)
+    {
+        _chart1.ShowTooltip(args.MouseX, args.MouseY, false);       
+    }
+    public void OnMouseLeaveChart1()
+    {
+        _chart2.HideTooltip();
+    }
+    public void OnMouseLeaveChart2()
+    {
+        _chart1.HideTooltip();     
+
+    }
+
+    public void OnMouseLeaveChart1(ChartMouseEventArgs args)
+    {
+            isTapHold = false;
+            isTouch = false;
+            isMouseMoveUpDetected = true;
+            _chart2.HideTooltip();            
+
+    }
+    public void OnMouseLeaveChart2(ChartMouseEventArgs args)
+    {
+
+            isTapHold = false;
+            isTouch = false;
+            isMouseMoveUpDetected = true;
+            _chart1.HideTooltip();            
+
+    }
+}
+
+```
+
+![Tooltip Synchronization in Blazor Chart](../images/blazor-chart-tooltip-synchronization.gif)
+
+## Crosshair synchronization
+
+The crosshair can be synchronized across multiple charts using the `ShowCrosshair` and `HideCrosshair` methods. When we hover over one chart, we call the showCrosshair method for the other charts to align with data points in other connected charts, simplifying data comparison and analysis.
+
+In the showCrosshair method, specify the following parameters programmatically to enable crosshair for a particular chart:
+
+* x - x-value of point or x-coordinate value.
+* y - y-value of point or y-coordinate value.
+
+```cshtml 
+ 
+@using Syncfusion.Blazor.Charts;
+@using Syncfusion.Blazor
+@using System.Net.Http.Json
+@inject NavigationManager NavigationManager
+@inject HttpClient Http
+
+<div class="row">
```

Comment:
In the tooltip topics, the code snippet contains codes related to zooming and tooltip. Could you please clean it up?